tests/transform-field-names/transform-field-names.t

```diff
@@ -0,0 +1,26 @@
+Verify behavior of `transform-field-names`
+
+If the `--field-map` includes a old field name that is the same as the new field
+name, this should be no-op.
+
+  $ echo '{"strain": "A"}' \
+  >  | $TESTDIR/../../transform-field-names \
+  >      --field-map "strain=strain"
+  {"strain":"A"}
+
+If the `--field-map` overwrites an existing field, then skip renaming and
+print a loud warning.
+
+  $ echo '{"strain": "A", "isolate": "B"}' \
+  >  | $TESTDIR/../../transform-field-names \
+  >      --field-map "isolate=strain"
+  WARNING: skipping rename of isolate because record already has a field named strain.
+  {"strain":"A","isolate":"B"}
+
+If the `--field-map` overwrites an existing field if using `--force` flag.
```

Comment:
minor non-blocking nit-pick: "The `--field-map` may overwrite an existing field ..."

Reply:
Updated in [force-push](https://github.com/nextstrain/ingest/compare/0f00c521ba2700726e60554b2c2f5194057e91ee..1f9a3822edf5dffabc68f21a93b44418cae88f28)